__tests__/integration/integration.test.ts

```diff
@@ -0,0 +1,232 @@
+      MultiTenancyMode: TenantedDeploymentMode.TenantedOrUntenanted,
+      RunRetentionPolicy: {
+        QuantityToKeep: 1,
+        ShouldKeepForever: false
+      }
+    })
+    globalCleanup.add(async () => repository.runbooks.del(runbook))
+    const runbookProcess = await repository.runbookProcess.get(
+      runbook.RunbookProcessId,
+      undefined
+    )
+    runbookProcess.Steps = [
+      {
+        Condition: RunCondition.Success,
+        Links: {},
+        PackageRequirement: PackageRequirement.LetOctopusDecide,
+        StartTrigger: StartTrigger.StartAfterPrevious,
+        Id: '',
+        Name: `Run a Script`,
+        Properties: {'Octopus.Action.TargetRoles': 'deploy'},
+        Actions: [
+          {
+            Id: '',
+            Name: 'Run a Script',
+            ActionType: 'Octopus.Script',
+            Notes: null,
+            IsDisabled: false,
+            WorkerPoolId: null,
+            Container: {
+              Image: null,
+              FeedId: null
+            },
+            WorkerPoolVariable: null,
+            Environments: [],
+            ExcludedEnvironments: [],
+            Channels: [],
+            TenantTags: [],
+            CanBeUsedForProjectVersioning: false,
+            Packages: [],
+            Properties: {
+              'Octopus.Action.RunOnServer': 'false',
+              'Octopus.Action.Script.ScriptSource': 'Inline',
+              'Octopus.Action.Script.Syntax': 'Bash',
+              'Octopus.Action.Script.ScriptBody': "ehco 'hello'"
+            },
+            Links: {},
+            IsRequired: false
+          }
+        ]
+      }
+    ]
+    await repository.runbookProcess.save(runbookProcess)
+    let snapshot = {} as RunbookSnapshotResource
+    snapshot.ProjectId = project.Id
+    snapshot.Name = `Snapshot${runId}`
+    snapshot.RunbookId = runbook.Id
+    snapshot = await repository.runbookSnapshots.create(snapshot, {
+      publish: true
+    })
+    standardInput.runbook = runbook.Id
+    const env = await repository.environments.create({
+      Name: `Test-${runId}`
+    })
+    globalCleanup.add(async () => repository.environments.del(env))
+    globalCleanup.add(async () => new Promise(r => setTimeout(r, 2500)))
+    standardInput.environments = env.Id
+  })
+
+  afterAll(async () => {
+    if (process.env.GITHUB_ACTIONS) {
+      setOutput('gha_selftest_project_name', standardInput.project)
+      setOutput('gha_selftest_environments', standardInput.environments)
+      setOutput('gha_selftest_runbook', standardInput.runbook)
+    } else {
+      await globalCleanup.cleanup()
+    }
+  })
+
+  test('can run runbook', async () => {
+    const messages: string[] = []
+    const w = new OctopusCliWrapper(
+      standardInput,
+      {},
+      m => messages.push(m),
+      m => messages.push(m)
+    )
+    await w.runRunbook(octoExecutable)
+    console.log('Got: ', messages)
+    // expectMatchAll(messages, [
```

Comment:
Suggest removing this commented-out code and replace with a comment explaining that the output is variable and we can't really assert on it